### DISCUSSION TITLE
next phase of jbang world dominiation....

### DISCUSSION THREAD

**Discussion by maxandersen** [jbangdev/jbang#337] [General]
...or something like that - lets aim high :) 

With latest release(s) over the weekend jbang now has the basics for writing using multiple files which been the most asked for request since day one. 

I waited this long as I wanted to see how far I could push jbang before multiple files was really needed - I still think lots of automation and tools will be just fine using single script source files; but with Quarkus backed apps it does gets much more interesting and practical to be able to use multiple sources.

There are still things that needs fixing/cleaned up but the basics are in place and with 0.47 can now be used to write full fledged apps with jbang.

Now, this means jbang.dev focus/wording is out of date and could need a refresh - looking for ideas/inspiration on what to put in there.

Any suggestions on what could be on jbang.dev and would have convinced you faster to try out jbang very welcome.

**Comment by abelsromero** [comment]
I suspect next thing people may be interested in is testing.

**Reply by maxandersen** [reply]
now we have multiple sources you should technically be able to just have a test.java with a `//SOURCES myapp.java` and then "boot" junit in test.java main...but for that to be really nice should get added support for `//DEPS` to be honored in the external sources..otherwise you have to duplicate them.

...but maybe we could make `jbang org.junit.platform:junit-platform-console-standalone:1.7.0` work in some way.
